pyproject.toml

```
@@ -0,0 +1,63 @@
+[build-system]
+requires = ["setuptools>=63.0"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+platforms = ["Linux x86, x86-64"]
+
+[tool.setuptools.packages.find]
+include = ["sopel", "sopel.*"]
+namespaces = false
+
+[project]
+name = "sopel"
+version = "8.0.0.dev0"
+description = "Simple and extensible IRC bot"
+authors = [
+  { name="dgw", email="dgw@technobabbl.es" },
+]
+readme = "README.rst"
+license = { file="COPYING" }
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Intended Audience :: System Administrators",
+    "License :: Eiffel Forum License (EFL)",
+    "License :: OSI Approved :: Eiffel Forum License",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Topic :: Communications :: Chat :: Internet Relay Chat",
+]
+requires-python = ">=3.7"
+dependencies = [
+    "xmltodict==0.12",
+    "pytz",
+    "praw>=4.0.0,<8.0.0",
+    "geoip2>=4.0,<5.0",
+    "requests>=2.24.0,<3.0.0",
+    "dnspython<3.0",
+    "sqlalchemy>=1.4,<1.5",
+    "importlib_metadata>=3.6",
+    "packaging",
+]
+
+[project.urls]
+"Homepage" = "https://sopel.chat/"
+"Release notes" = "https://github.com/sopel-irc/sopel/releases"
+"Documentation" = "https://sopel.chat/docs/"
+"Bug Tracker" = "https://github.com/sopel-irc/sopel/issues"
+"Donate" = "https://opencollective.com/sopel"
+"Github Sponsor" = "https://github.com/sponsors/sopel-irc"
```

```suggestion
"Funding on Open Collective" = "https://opencollective.com/sopel"
"Sponsor on GitHub" = "https://github.com/sponsors/sopel-irc"
```

This will use the correct type of icon on PyPI for both links. It's chosen by matching what the string starts with (I know, I know, not my architectural decision).

It also differentiates the two a bit. The primary purpose of OC is public expenditure records, while all of our current sponsors do so via GH.